### PR TITLE
Skip to content on home page

### PR DESCRIPTION
**Pull request by rmrose78**
To fix a bug on a prior issue with the Skip-to-main link being focusable on all pages while only working on the home page. The Link is now only on the home page. Issue #346 will be worked on to have a link work to skip-to-main content to each specific page.

## Description
On the Home page press the tab key twice to focus and unhide the skip-to-main link. click or hit enter on the link to be taken to #our_stories content.

## Related Issue
This branch was created to address a bug created on branch "RRose/AccessibilitySkip" when adding the skip-to-main link.

## Motivation and Context
The skip-to-main link created in "RRose/AccessibilitySkip" branch was accessible on all pages while only working on the home page. Now the link is only accessible on the home page. future PR will expand link to work on all pages.

## How Has This Been Tested?
Fix was tested visually and functionally with "npm run dev" as well as passed tests through "npm run test".

## Screenshots (if appropriate):

Screen Shot 2022-04-15 at 11 47 58 AM

## Types of changes

- [ x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: tests/components/Layout.test.js [hunk 1]
```diff
@@ -1,7 +1,10 @@
 import { render } from '@testing-library/react'
 import TemplateWrapper from '@/components/Layout'
+import * as nextRouter from 'next/router'
 
 describe('<Layout />', () => {
+  nextRouter.useRouter = jest.fn() // eslint-disable-line
```
Comment (andrew-t-james) [inline tests/components/Layout.test.js:6]: Please use jest's module mocks for this.

**Review by andrew-t-james — CHANGES_REQUESTED** [review]

File: tests/components/Layout.test.js (hunk 1 above)
Comment (rmrose78) [reply tests/components/Layout.test.js:6]: I will make sure I read into the Jest Module for updating tests in the future.

Comment (andrew-t-james) [inline tests/components/Layout.test.js:6]: Please move over to jest module mocks. This current set has potential to cause unexpected side effects when running tests. See example [here][
(https://github.com/Vets-Who-Code/vets-who-code-app/blob/master/tests/setup-test-env.js#L10]
)

File: tests/components/Nav.test.js [hunk 2]
```diff
@@ -1,7 +1,10 @@
 import { render } from '@testing-library/react'
 import Nav from '@/components/Nav'
+import * as nextRouter from 'next/router'
 
 describe('<Nav />', () => {
+  nextRouter.useRouter = jest.fn() // eslint-disable-line
```
Comment (andrew-t-james) [inline tests/components/Nav.test.js:6]: Please move over to jest module mocks. This current set has potential to cause unexpected side effects when running tests. See example [here](https://github.com/Vets-Who-Code/vets-who-code-app/blob/master/tests/setup-test-env.js#L10]
)

**Review by andrew-t-james — CHANGES_REQUESTED** [review]

**Pull request closed without merging**